Snakefile

```diff
@@ -24,7 +24,15 @@ rule prepare_example:
     shell:
         "cp -r examples/{wildcards.scenario}/preprocessed results/{wildcards.scenario}/preprocessed"
 
-
+rule prepare_conv_pp:
+    input:
+        opsd="raw/conventional_power_plants_DE.csv",
+	    gpkg="raw/boundaries_germany_nuts3.gpkg"
```

Comment:
```suggestion
	    gpkg="raw/boundaries_germany_nuts3.gpkg",
	    script="scripts/prepare_conv_pp.py"
```

Reply:
This tells snakemake to track changes in the script and rerun if the script has changed.